src/components/CommentCard.jsx

```diff
@@ -12,6 +34,13 @@ function CommentCard({ comments, article }) {
             <h3>By {comment.author}</h3>
             <h4>Votes {comment.votes}</h4>
             <h4>{comment.created_at}</h4>
+            <button
```

Comment:
Rather than have a user guess which comment they can delete as they all have a delete button, what about conditionally rendering this button so that it only appears on comments I am allowed to delete. Else your error about not yours to delete shows on EVERY comment. This way you don't have to worry about that particular error, as you can only see the button if the author matches the 'signed in' person.